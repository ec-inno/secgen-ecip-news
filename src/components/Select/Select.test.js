import React from 'react';
import renderer from 'react-test-renderer';
import Select from './Select';

describe('Select', () => {
  it('renders correctly', () => {
    const props = { id: 'foo' };
    const tree = renderer.create(<Select {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
