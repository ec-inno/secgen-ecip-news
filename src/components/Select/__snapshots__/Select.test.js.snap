// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renders correctly 1`] = `
<div
  className="ecl-form-group ecl-form-group--select"
>
  
  
  <div
    className="ecl-select__container"
  >
    <select
      className="ecl-select"
      disabled={false}
      id="foo"
    />
    <div
      className="ecl-select__icon"
    >
      <svg
        aria-hidden="true"
        className="ecl-select__icon-shape ecl-icon ecl-icon--s ecl-icon--rotate-180"
        focusable="false"
      >
        
        
        <use
          xlinkHref="file#ui--corner-arrow"
        />
      </svg>
    </div>
  </div>
</div>
`;
